### PR TITLE
Create a todo list structure

### PR DESCRIPTION
**Pull request by fuadmoin**
I added html css and javascript in src folder and used webpack to bundle them. And inside index.js I created the structure for the todo list. The display function will display based on the order of index.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
   
   <img src="./img/logo2.png" alt="logo" width="140"  height="auto" />
   <br/>
-
```
Comment (zieeco) [inline README.md]: ### CONTRIBUTING SECTION

- [ ] Please, kindly add the link to the Github issues page of this project to make the users quickly navigate to the issue page.

File: src/style.css [hunk 2]
```diff
@@ -77,3 +77,7 @@
   left: -20px;
   border-style: none;
 }
+
```
Comment (zieeco) [inline src/style.css:76]: ### COMMENTED CODE BLOCK

- [ ] Please, let's remove this commented [code](https://github.com/fuadmoin/TODOLIST/blob/9d0ef2bab6e9e84a55e6ad7f3a3241e107fcbb7d/src/style.css#L32) to keep our codebase clean.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @fuadmoin ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Good commit messages.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Professional README :100: 

## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @fuadmoin 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**